query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_delete/no_action.rs

```diff
@@ -45,22 +45,26 @@ mod one2one_req {
         Ok(())
     }
 
-    /// Deleting the parent leaves the data in a integrity-violating state.
+    /// Deleting the parent must fail if a child is connected.
     #[connector_test(only(MongoDb))]
```

Comment:
On most of the changed tests, you kept `only(MongoDb)` which means this test is _only_ ran on Mongo. I believe this test was initially only run on Mongo because SQL databases would eventually fail because of fk constraints. I suppose we want to run this test on all dbs now that NoAction is an alias to Restrict

Reply:
While the behavior of Mongo now resembles the other databases, both the error code and the error message are signficantly different in Mongo, hence why it's helpful to have a separate test. Would it be useful if I added more comments to highlight that fact?